docs/source/usage/dashboard.rst

```diff
@@ -8,7 +8,7 @@ It provides a graphical interface to a subset of ImpactX functionality.
 
 .. note::
 
-   ImpactX Dashboard is provided as a preview and continues to be developed.
+   ImpactX Dashboard is provided as a preview and continues to be developed.  As a result, it may not yet include all the features available in ImpactX.
```

Comment:
```suggestion
   ImpactX Dashboard is provided as a preview and continues to be developed.
   As a result, it may not yet include all the features available in ImpactX.
```